cli/src/Types.ts

```diff
@@ -55,9 +55,9 @@ export enum WorkingGroups {
   Curators = 'curators',
   Forum = 'forum',
   Membership = 'membership',
-  OperationsAlpha = 'operationsAlpha',
-  OperationsBeta = 'operationsBeta',
-  OperationsGamma = 'operationsGamma',
+  Builders = 'builders',
```

Comment:
I don't think it is safe to change these, they should match the runtime name of the group?

Reply:
They can be customized, since CLI uses mappings like `apiModuleByGroup`

Reply:
Admittedly, I didn't test more than a couple groups after the change, but `operationsAlpha` isn't the runtime name of the group either.  I think these are just "shorts" used for the users convenience, whereas the actual runtime reference are from `cli/src/Api.ts`.